src/main/java/org/scalameter/japi/annotation/setupBeforeAll.java

```diff
@@ -0,0 +1,18 @@
+package org.scalameter.japi.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/** Optional annotation that points to a
+ *  setupBeforeAll method for a benchmark method.
```

Comment:
Using backticks around `setupBeforeAll` will ensure that it is rendered nicely in ScalaDoc.